segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -171,7 +173,7 @@ public ChunkedSegmentStorage(ChunkStorage chunkStorage, ChunkMetadataStore metad
      * @param containerId   container id.
      * @throws Exception In case of any errors.
      */
-    public void bootstrap(int containerId, ChunkMetadataStore metadataStore) throws Exception {
+    public CompletableFuture<Void> bootstrap(int containerId, ChunkMetadataStore metadataStore) throws Exception {
```

Comment:
This has become the biggest class in the whole project. Can we do something to make it smaller? Can we move each of those operations in their own, separate files?

Reply:
now I have split big bad class into smaller classes.